openapi.yml

```diff
@@ -1808,6 +1808,122 @@ paths:
+        '201':
+          content:
+            application/json:
+              schema:
+                $ref: ./models/InviteDetail.v1.yaml
+              examples:
+                default:
+                  value:
+                    id: 93
+                    company: some text
+                    created_at: '2018-02-10T09:30Z'
+                    email: some text
+                    end_date: '2018-02-10T09:30Z'
+                    first_name: some text
+                    last_name: some text
+                    start_date: '2018-02-10T09:30Z'
+                    hosts:
+                      - id: 23
+                        email: some text
+                        first_name: some text
+                        last_name: some text
+                        mobile_number: some text
+                        profile_pic_url: some text
+                      - id: 33
+                        email: some text
+                        first_name: some text
+                        last_name: some text
+                        mobile_number: some text
+                        profile_pic_url: some text
+                    location:
+                      id: 20
+                      name: some text
+                    watchlist_colour: YELLOW
+                    template:
+                      id: 15
+                      name: some text
+                      template_type: some text
+                    custom_fields:
+                      - format: string
+                        field_name: some text
+                        field_value: some text
+                      - format: string
+                        field_name: some text
+                        field_value: some text
+                    notification_triggers:
+                      - offset_direction: AFTER
+                        offset_amount: 66
+                        offset_origin: END
+                        email_template_id: 80
+                        notification_groups:
+                          - some text
+                          - some text
+                        offset_unit: hours
+                        email_template_name: some text
+                      - offset_direction: BEFORE
+                        offset_amount: 21
+                        offset_origin: END
+                        email_template_id: 30
+                        notification_groups:
+                          - some text
+                          - some text
+                        offset_unit: days
+                        email_template_name: some text
+                    mobile_number: some text
+                    invite_watchlist:
+                      id: 3
+                      external_colours:
+                        - some text
+                      internal_colours:
+                        - some text
+                        - some text
+          description: Created an `Invite` from the given `Registration` UUID.
+        '400':
```

Comment:
@stevenallen05 were you doing some weird `4xx` thing for these now?
I remember it seemed that Shabini just had to do a fix for her added paths not including responses for the other 4xx responses other than 400.

Reply:
It's not mandatory to do it either way. It just saves on the copy/paste to do the `4XX` wildcard.